docs/docs/llms/flow-policy.mdx

```diff
@@ -3,9 +3,179 @@ id: flow-policy
 sidebar_label: Flow Policy
 title: Flow Policy
 abstract: |
-  Running Flows with the FlowPolicy.
+  Guide on implementing and managing Flows with the FlowPolicy.
 ---
 
 import RasaDiscoveryBanner from "@theme/RasaDiscoveryBanner";
+import flowStackEvolution from "./flow_stack_evolution.png";
+import flowStackOverview from "./flow_stack.png";
 
-<RasaDiscoveryBanner/>
+<RasaDiscoveryBanner />
+
+The Flow Policy in Rasa is a state machine that enables you to
+manage conversational flows in your chatbot and combine your business logic 
+with machine learning. 
+
+The policy manages the state of your bot, transitioning between states, and
+initiating new flows when required. The "Flow Stack" represents 
```

Comment:
Do we want to explain that the flow stack is LIFO? I feel like most developers know what a stack is, but not assuming prior knowledge is one of our documentation principles. Maybe something for later though. Up to you

Reply:
will do so further down :+1: